test/job/Job.spec.ts

```diff
@@ -1,18 +1,40 @@
-import { toJob } from '../../src/job/Job';
+import { toCronJob, toIntervalJob } from '../../src/job/Job';
 import { MomoJob } from '../../src';
 
-describe('toJob', () => {
-  it('sets defaults', () => {
-    const job: MomoJob = {
-      name: 'test',
-      schedule: { interval: '1 second', firstRunAfter: 0 },
-      handler: () => undefined,
-    };
+describe('Job', () => {
+  describe('toCronJob', () => {
+    it('sets defaults', () => {
+      const job: MomoJob = {
+        name: 'test',
+        schedule: { cronSchedule: '0 9 * * 1-5' },
+        handler: () => undefined,
+      };
 
-    expect(toJob(job)).toEqual({
-      ...job,
-      concurrency: 1,
-      maxRunning: 0,
+      expect(toCronJob(job)).toEqual({
+        ...job,
+        schedule: { cronSchedule: '0 9 * * 1-5' },
+        concurrency: 1,
+        maxRunning: 0,
+      });
     });
   });
+
+  describe('toIntervalJob', () => {
+    it('sets defaults', () => {
+      const job: MomoJob = {
+        name: 'test',
+        schedule: { interval: '1 second' },
+        handler: () => undefined,
+      };
+
+      expect(toIntervalJob(job)).toEqual({
+        ...job,
+        schedule: { interval: '1 second', parsedInterval: 1000, firstRunAfter: 0, parsedFirstRunAfter: 0 },
+        concurrency: 1,
+        maxRunning: 0,
+      });
+    });
+
+    // TODO add more test cases
```

Comment:
TODO
missing tests for:
 setting default of firstRunAfter
parsing firstRunAfter